tests/gold_tests/logging/log_pipe.test.py

```diff
@@ -0,0 +1,190 @@
+'''
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+
+Test.Summary = '''
+Test custom log file format
+'''
+# This test requires Curl and is Linux specific.
+Test.SkipUnless(
+    Condition.HasProgram(
+        "curl", "Curl need to be installed on system for this test to work"),
```

Comment:
As I recall they were removing the curl checks.  So we can assume any testing system has curl.

Reply:
This was part of 6c1a90f4f209a02cf1345ac4c20e2e67c4c58db1:
`The checks for grep, sed, curl and nc are superflous. Most systems will have them,
    and if not, we should just fail hard, and it's obvious what the user has to do.
`

Reply:
Thanks for pointing this out, @shinrich and @randall . I updated the PR accordingly.